### PR TITLE
LPS-XXXXX Adding first test

### PR DESCRIPTION
**Pull request by weller0246**
Test pull request

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/HelloPoshi.testcase [hunk 1]
```diff
@@ -0,0 +1,23 @@
+definition {
+  property testray.main.component.name = "Blogs";
+
+  setUp {
+    User.firstLoginPG();
+  }
+
+  test HelloWorld {
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/HelloPoshi.testcase:8]: Cool name !

**Review by gaborlovasdotcom — APPROVED** [review]
LGTM

**Comment by weller0246** [comment]
ci:help

**Comment by weller0246** [comment]
ci:test:sf

**Comment by weller0246** [comment]
ci:test:sf

**Comment by weller0246** [comment]
ci:test:blogs:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:stop

**Comment by gaborlovasdotcom** [comment]
Stopping test run, as we will modify PQL to run this new test only.

**Comment by weller0246** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:close

**Pull request closed without merging**